src/gui/guiTable.cpp

```diff
@@ -62,12 +61,12 @@ GUITable::GUITable(gui::IGUIEnvironment *env,
 	}
 
 	const s32 s = skin->getSize(gui::EGDS_SCROLLBAR_SIZE);
-	m_scrollbar = Environment->addScrollBar(false,
-			core::rect<s32>(RelativeRect.getWidth() - s,
-					0,
-					RelativeRect.getWidth(),
-					RelativeRect.getHeight()),
-			this, -1);
+	m_scrollbar = new guiScrollBar(Environment, this, -1,
+			core::rect<s32>(RelativeRect.getWidth() - s, 0,
+							   RelativeRect.getWidth(),
+							   RelativeRect.getHeight()),
```

Comment:
Why is here so much indentation? And why are there spaces?!
It looks like you have tried to align this.

Reply:
I believe this was the work for clang-format, however, I have since realised that my IDE (QT Creator) is set up to force tabs for indentation and the 'beautifier' does not seem to recognise some of MTs format rules.

it appears this file in particular is already whitelisted so I will just restore the original indentation here.